.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "airbnb",
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true
+  },
+  "globals": {
+    "Clipboard": true,
+    "c3": true,
+    "d3": true,
+    "analytics": true
+  },
+  "rules": {
+    "no-console": "off",
+    "strict": "off",
+    "no-script-url": "off"
+  },
+  "settings": {
+    "import/resolver": {
+      "webpack": {
+        "config": "webpack.config.js"
+      }
+    }
+  }
+}
```

Comment:
similarly remove these - eslintrc configuration will get standardized across all Carbon repos in a different PR